support/k8s/k8s-workload-registrar/controller_test.go

```diff
@@ -281,6 +282,73 @@ func TestControllerLabelBasedRegistrationIgnoresPodsWithoutLabel(t *testing.T) {
 	require.Len(t, r.GetEntries(), 0)
 }
 
+func TestPodSpiffeId(t *testing.T) {
+	for _, test := range []struct {
+		expectedSpiffeID  string
+		configLabel       string
+		podLabel          string
+		configAnnotation  string
+		podAnnotation     string
+		podNamespace      string
+		podServiceAccount string
+	}{
+		{
```

Comment:
nit: if this would be structured using a `t.Run` call, it could report multiple failures in case that two or more test cases fail (current implementation reports the first failure only).
Here is an example of using `t.Run` in the way that I'm referring to: https://github.com/spiffe/spire/blob/master/cmd/spire-server/cli/x509/mint_test.go#L189

Reply:
It also helps avoid accidental contamination of state between different test cases :)
PR updated